mod_reforged/hooks/skills/perks/perk_nine_lives.nut

```diff
@@ -0,0 +1,40 @@
+::mods_hookExactClass("skills/perks/perk_nine_lives", function(o) {
+    local create = o.create;
+	o.create = function()
+	{
+        create();
+
+        // New Variables to control and tweak the behavior of this perk
+        this.m.MinHP <- 11;
+        this.m.MaxHP <- 15;
+
+        // This makes the perk now always show by default on top if characters while it is not procced yet
+		this.m.IconMini = "perk_07_mini";
+		this.m.Overlay = "perk_07";
+        this.m.Type = (this.m.Type | this.Const.SkillType.StatusEffect);    // Therefor it needs the StatusEffect type
+    }
+
+    // Nine lives now shows up as an effect while available. Therefor we give it a custom more detailed description
+	o.getDescription = function()
+	{
+		return "Upon receiving the next killing blow, this character will survive with " + ::MSU.Text.colorGreen(this.m.MinHP + "-" + this.m.MaxHP) + " Hitpoints, all damage over time effects are cured and they will gain a stat buff until their next turn.";
```

Comment:
Improved wording: `Upon receiving the next killing blow, this character will survive with <calculation> Hitpoints, all damage over time effects will be removed, and Melee Skill, Melee Defense, Resolve and Initiative will be increased until the start of their next turn.`